src/Directory.Build.props

```diff
@@ -0,0 +1,10 @@
+<Project ToolsVersion="15.0">
+    <PropertyGroup>
+        <PublishRepositoryUrl>true</PublishRepositoryUrl>
+        <EmbedUntrackedSources>true</EmbedUntrackedSources>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+        <DebugType>embedded</DebugType>
```

Comment:
this embeds all information in the pdb into the base dll, for easier packaging. larger file sizes, but we don't really care about that.